Seed.Api/Controllers/UserController.cs

```diff
@@ -42,7 +42,7 @@ public async Task<IActionResult> GetAll()
         /// <param name="userId" cref="Guid">Guid of the user</param>
         /// <response code="200">User for the given id</response>
         /// <response code="404">User not found</response>
-        [HttpGet("{userId}")]
+        [HttpGet("ById/{userId}")]
```

Comment:
This is not a restfull address remove ById

Reply:
Removed